source/load-balancers/tests/lb-connection-establishment.yml

```diff
@@ -0,0 +1,29 @@
+description: connection establishment for load-balanced clusters
+
+schemaVersion: '1.3'
+
+runOnRequirements:
+  - topologies: [ load-balanced ]
+
+createEntities:
+  - client:
+      id: &client0 client0
+      uriOptions:
+        # Explicitly set loadBalanced to false to override the option from the global URI.
+        loadBalanced: false
+  - database:
+      id: &database0 database0
+      client: *client0
+      databaseName: &database0Name database0
+
+tests:
+  - description: operations against load balancers fail if URI contains loadBalanced=false
+    skipReason: servers have not implemented LB support yet so they will not fail the connection handshake in this case
+    operations:
+      - name: runCommand
+        object: *database0
+        arguments:
+          commandName: ping
+          command: { ping: 1 }
+        expectError:
+          isClientError: false
```

Comment:
I'm not sure I understand. This is asserting that the error is _not_ a client error because it's caused by the server. Also, small clarification: in LB mode, there is no server discovery. We assume that the load balancer is always selectable, so there are no server selection timeout errors either. I added an `expectEvents` block to assert that no events are sent because the connection handshake fails, so we never send the `ping` command.

Reply:
Ignore my last comment. I must have misread this as `isClientError: true` when I wrote that.